py_lesson3_flask/templates/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Регистрация</title>
+</head>
+<body>
+    <p>{{ message }}</p>
+    <form action="" method="post">
+        {{ form.hidden_tag() }}
+        {{ form.csrf_token }}
+        <p>
+            {{ form.name.label }}<br>
+            {{ form.name(class="form-control") }}<br>
+            {% for error in form.name.errors %}
+                <p class="alert alert-danger" role="alert">{{ error }}</p>
+            {% endfor %}
+        </p>
+        <p>
+            {{ form.last_name.label }}<br>
+            {{ form.last_name(class="form-control") }}<br>
+            {% for error in form.last_name.errors %}
+                <p class="alert alert-danger" role="alert">{{ error }}</p>
+            {% endfor %}
+        </p>
+        <p>
+            {{ form.email.label }}<br>
+            {{ form.email(class="form-control") }}<br>
+            {% for error in form.email.errors %}
+                <p class="alert alert-danger" role="alert">{{ error }}</p>
+            {% endfor %}
+        </p>
+        <p>
+            {{ form.password.label }}<br>
+            {{ form.password(class="form-control", type="password") }}<br>
+            {% for error in form.password.errors %}
+                <p class="alert alert-danger" role="alert">{{ error }}</p>
+            {% endfor %}
+        </p>
+
+        <p>{{ form.submit(type="submit", class="btn btn-primary") }}</p>
```

Comment:
Очень много повторений кода.
```suggestion
        {{ form.hidden_tag() }}
        {{ form.csrf_token }}
        {% for field in form if field.name != 'csrf_token' %}
        <p>
            {{ field.label }}<br>
            {{ field(class="form-control") }}<br>
            {% for error in field.errors %}
                <p class="alert alert-danger" role="alert">{{ error }}</p>
            {% endfor %}
        </p>
        {% endfor %}
        <p>{{ form.submit(type="submit", class="btn btn-primary") }}</p>
```